# all numbers currently random

position_tolerance: 0.02 # meters, if fourbar position is +/- than the server set success

joint: four_bar

# distance that the server should be called with

# purposly capitlized, same terminolgy in the elevater
# all XXX_below is if the service call to the fourbar should have the "below" field set to
# all VREY wrong, need to be angles
cube:
  intake: 0.3
  low_node: 0.6
  middle_node: 0.5
  high_node: 0.55

vertical_cone:
  intake: 0.3
  low_node: 0.6
  middle_node: 0.5
  high_node: 0.5

base_towards_us_cone:
  intake: 0.4
  low_node: 0.3
  middle_node: 0.25
  high_node: 0.5

base_away_us_cone:
  intake: 0.3
  low_node: 0.6
  middle_node: 0.5
  high_node: 0.5

# minimum distance of fourbar
# will clamp angles within this amount
# there should be an overlap between all three states, if not that is bad
safety_high:
  min_angle: 0.0
  max_angle: 1.57

safety_mid:
  min_angle: 0.6
  max_angle: 0.9

safety_intake:
  min_angle: 0.7
  max_angle: 0.8